packages/xo-server/src/api/xostor.mjs

```diff
@@ -191,17 +192,52 @@ export const create = defer(async function (
     })
 
     const hosts = hostIds.map(hostId => this.getObject(hostId, 'host'))
+
     if (!hosts.every(host => host.$pool === hosts[0].$pool)) {
       // we need to do this test to ensure it won't create a partial LV group with only the host of the pool of the first master
       throw new Error('All hosts must be in the same pool')
     }
 
-    const boundInstallDependencies = installDependencies.bind(this)
-    await asyncEach(hosts, host => boundInstallDependencies({ host }), { stopOnError: false })
-
     const host = hosts[0]
     const xapi = this.getXapi(host)
 
+    const handleHostsDependencies = defer(async ($defer, hosts) => {
+      const boundInstallDependencies = installDependencies.bind(this)
+      const boundUpdateDependencies = updateDependencies.bind(this)
+      const pool = Object.values(xapi.objects.indexes.type.pool)[0]
+
+      await asyncEach(
```

Comment:
default asyncEach concurercny is 10, is it ok with the installation process ?

Reply:
Why wouldn't it be? 🤔

Reply:
I think  a pool with xostore is limited to a 7 hosts for now, and was waondering if the async each is here to be future proof, or because  we need to limit the tool stack concurrency drastically